app/views/options/index.html.haml

```diff
@@ -41,6 +41,28 @@
         quick_donation_redirect |
         donation_ack_from |
         ) |
+      .form-row
+        .col-md-4.text-right
+          %label.col-form-label Allow Monthly Recurring Donations
+          = option_description_for(:allow_recurring_donations)
+        .col-md-8{id:"allow_recurring_donations_select"}
+          = f.select :allow_recurring_donations, [['Yes', true], ['No', false]], {}, {:class => 'form-control'}
+      - if Option.allow_recurring_donations
+        .form-row{id:"default_donation_type_form_row"}
+          .col-md-4.text-right
+            %label.col-form-label Default Donation Type
+            = option_description_for(:default_donation_type)
+          .col-md-8{id:"donation_type_radio"}
+            .radio-group.form-inline
+              = f.radio_button :default_donation_type, 'one', :class => 'form-control', :id => 'one'
```

Comment:
i don't lke the use of the string constant 'one' because it's not clear what other values it coudl legally have.  as above: i suggest you define constants **within the Donation or RecuringDonation class** for the possible values of donations as one-time or monthly; then refer to those constants in the controller as well as in the view, so that there is DRYness around how those two possible values are represnted.